lib/graphlient/errors/faraday_server_error.rb

```diff
@@ -4,8 +4,8 @@ class FaradayServerError < ServerError
       def initialize(inner_exception)
         super(inner_exception.message, inner_exception)
         @inner_exception = inner_exception
-        @response = inner_exception.response[:body]
-        @status_code = inner_exception.response[:status]
+        @response = inner_exception.response.try(:[],:body)
+        @status_code = inner_exception.response.try(:[],:status)
```

Comment:
The `#try` method is a method that[ ActiveSupport defines](https://api.rubyonrails.org/classes/Object.html#method-i-try) on almost all objects and we don't want to depend on it.